sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobAsyncClientBase.java

```diff
@@ -1040,29 +1040,10 @@ Mono<Response<String>> copyFromUrlWithResponse(BlobCopyFromUrlOptions options, C
      * Reads the entire blob. Uploading data must be done from the {@link BlockBlobClient}, {@link PageBlobClient}, or
      * {@link AppendBlobClient}.
      *
-     * <p><strong>Code Samples</strong></p>
-     *
-     * <!-- src_embed com.azure.storage.blob.specialized.BlobAsyncClientBase.download -->
-     * <pre>
-     * ByteArrayOutputStream downloadData = new ByteArrayOutputStream&#40;&#41;;
-     * client.download&#40;&#41;.subscribe&#40;piece -&gt; &#123;
-     *     try &#123;
-     *         downloadData.write&#40;piece.array&#40;&#41;&#41;;
-     *     &#125; catch &#40;IOException ex&#41; &#123;
-     *         throw new UncheckedIOException&#40;ex&#41;;
-     *     &#125;
-     * &#125;&#41;;
-     * </pre>
-     * <!-- end com.azure.storage.blob.specialized.BlobAsyncClientBase.download -->
-     *
-     * <p>For more information, see the
-     * <a href="https://docs.microsoft.com/rest/api/storageservices/get-blob">Azure Docs</a></p>
-     *
-     * <p>This method will be deprecated in the future. Use {@link #downloadStream()} instead.
-     *
      * @return A reactive response containing the blob data.
+     * @deprecated use {@link #downloadStream()} instead.
      */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
+    @Deprecated
```

Comment:
I'd continue to have the `@ServiceMethod` annotation even though this has been deprecated. Same comment applies to the other changes like this